README.md

```diff
@@ -1,18 +1,47 @@
 # Salus - SBOM Tool
 
+[![Build](https://github.com/microsoft/sbom-tool/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/microsoft/sbom-tool/actions/workflows/build.yml)
+![GitHub release (latest by date)](https://img.shields.io/github/downloads/microsoft/sbom-tool/latest/total)
+![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/microsoft/sbom-tool?include_prereleases)
+
 ## Introduction
 
 The SBOM tool is a highly scalable and enterprise ready tool to create SPDX 2.2 compatible SBOMs for any variety of artifacts.
 
 ## Table of Contents
 
-* [Installation](docs/installation.md)
-* [Building and running Sbom tool](docs/build-and-run.md)
+* [Installation](#installation)
+* [Run the tool](#run-the-tool-to-generate-an-sbom)
+* [Building the tool from source](docs/build-and-run.md)
```

Comment:
Shouldn't this file content be on ReadMe? Instead of linking back to the docs folder?

If not, then this should not be in Table of Contents since it is not here.

Reply:
Fixed